packages/shorebird_cli/lib/src/artifact_manager.dart

```diff
@@ -16,6 +17,22 @@ final artifactManagerRef = create(ArtifactManager.new);
 ArtifactManager get artifactManager => read(artifactManagerRef);
 
 class ArtifactManager {
+  /// Extract the release version from an appbundle.
+  Future<String> extractReleaseVersionFromAppBundle(
+    String appBundlePath,
+  ) async {
+    await cache.updateAll();
+
+    final results = await Future.wait([
```

Comment:
```suggestion
    final [versionName, versionCode] = await Future.wait([
```

Reply:
Not sure I understand the rationale behind this suggestion. We just want an opaque string that identifies the release version.

Reply:
Just a small syntax sugar. With the suggestion you can get rid of lines 31 and 32

Reply:
Oh gotcha, nice! Done.